### ISSUE TITLE
Unable to access :8080

### ISSUE THREAD

**Issue by ajeetraina** [fnatte/pizza-tribes#1]
I tried running the docker-compose file but it complained: @fnatte 

```
$ docker-compose up -d
ERROR: Couldn't find env file: /root/pizza-tribes/.env
[node2] (local) root@192.168.0.7 ~/pizza-tribes
```

I do see there is .env.default file, so I renamed it to .env.
But still I am not able to access localhost:8080.

**Comment by ajeetraina** [comment]
Here is what how containers status look like:

```
docker ps
CONTAINER ID   IMAGE                                        COMMAND                  CREATED              STATUS              PORTS                                                                  NAMES
a83b67c4fb3b   redislabs/redismod:edge                      "redis-server --load…"   About a minute ago   Up About a minute   0.0.0.0:6379->6379/tcp, :::6379->6379/tcp                              pizza-tribes_redis_1
55e9a4b6f0e7   ghcr.io/fnatte/pizza-tribes-updater:latest   "/app/pizza-tribes-u…"   About a minute ago   Up About a minute   8080/tcp                                                               pizza-tribes_updater_1
ccec918bd53f   ghcr.io/fnatte/pizza-tribes-webapp:latest    "/docker-entrypoint.…"   About a minute ago   Up About a minute   80/tcp                                                                 pizza-tribes_webapp_1
07037ee1caee   redislabs/redisinsight                       "bash ./docker-entry…"   About a minute ago   Up About a minute   0.0.0.0:8001->8001/tcp, :::8001->8001/tcp                              pizza-tribes_redisinsight_1
ffa2d170db91   ghcr.io/fnatte/pizza-tribes-front:latest     "caddy run --config …"   About a minute ago   Up About a minute   80/tcp, 443/tcp, 2019/tcp, 0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   pizza-tribes_front_1
6ad7b04dbb23   ghcr.io/fnatte/pizza-tribes-worker:latest    "/app/pizza-tribes-w…"   About a minute ago   Up About a minute   8080/tcp 
```

**Comment by ajeetraina** [comment]
```
 docker-compose logs -f
front_1  | {"level":"info","ts":1626950285.4283724,"msg":"using provided configuration","config_file":"/etc/caddy/Caddyfile","config_adapter":"caddyfile"}
front_1  | {"level":"warn","ts":1626950285.4305885,"msg":"input is not formatted with 'caddy fmt'","adapter":"caddyfile","file":"/etc/caddy/Caddyfile","line":2}
front_1  | {"level":"info","ts":1626950285.431589,"logger":"admin","msg":"admin endpoint started","address":"tcp/localhost:2019","enforce_origin":false,"origins":["localhost:2019","[::1]:2019","127.0.0.1:2019"]}
front_1  | {"level":"info","ts":1626950285.4322782,"logger":"http","msg":"server is listening only on the HTTPS port but has no TLS connection policies; adding one to enable TLS","server_name":"srv0","https_port":443}
front_1  | {"level":"info","ts":1626950285.4323218,"logger":"http","msg":"enabling automatic HTTP->HTTPS redirects","server_name":"srv0"}
front_1  | {"level":"info","ts":1626950285.433036,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc000245260"}
front_1  | {"level":"info","ts":1626950285.4455874,"logger":"tls","msg":"cleaning storage unit","description":"FileStorage:/data/caddy"}
front_1  | {"level":"info","ts":1626950285.447305,"logger":"tls","msg":"finished cleaning storage units"}
front_1  | {"level":"warn","ts":1626950285.4908516,"logger":"pki.ca.local","msg":"installing root certificate (you might be prompted for password)","path":"storage:pki/authorities/local/root.crt"}
front_1  | 2021/07/22 10:38:05 Warning: "certutil" is not available, install "certutil" with "apt install libnss3-tools" or "yum install nss-tools" and try again
front_1  | 2021/07/22 10:38:05 define JAVA_HOME environment variable to use the Java trust
front_1  | 2021/07/22 10:38:05 certificate installed properly in linux trusts
front_1  | {"level":"info","ts":1626950285.549572,"logger":"http","msg":"enabling automatic TLS certificate management","domains":["192.168.1.9"]}
front_1  | {"level":"info","ts":1626950285.5506968,"msg":"autosaved config (load with --resume flag)","file":"/config/caddy/autosave.json"}
front_1  | {"level":"info","ts":1626950285.5509126,"msg":"serving initial configuration"}
front_1  | {"level":"info","ts":1626950285.5510662,"logger":"tls.obtain","msg":"acquiring lock","identifier":"192.168.1.9"}
front_1  | {"level":"info","ts":1626950285.5543234,"logger":"tls.obtain","msg":"lock acquired","identifier":"192.168.1.9"}
front_1  | {"level":"info","ts":1626950285.5575469,"logger":"tls.obtain","msg":"certificate obtained successfully","identifier":"192.168.1.9"}
front_1  | {"level":"info","ts":1626950285.5575945,"logger":"tls.obtain","msg":"releasing lock","identifier":"192.168.1.9"}
front_1  | {"level":"warn","ts":1626950285.5582843,"logger":"tls","msg":"stapling OCSP","error":"no OCSP stapling for [192.168.1.9]: no OCSP server specified in certificate"}
webapp_1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
webapp_1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
webapp_1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
webapp_1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
webapp_1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
webapp_1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
webapp_1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
webapp_1  | /docker-entrypoint.sh: Configuration complete; ready for start up
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: using the "epoll" event method
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: nginx/1.21.1
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: built by gcc 8.3.0 (Debian 8.3.0-6) 
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: OS: Linux 5.10.25-linuxkit
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: getrlimit(RLIMIT_NOFILE): 1048576:1048576
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker processes
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 32
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 33
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 34
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 35
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 36
webapp_1  | 2021/07/22 10:38:06 [notice] 1#1: start worker process 37
migrator_1  | {"level":"info","time":"2021-07-22T10:38:06Z","message":"Starting migrator"}
migrator_1  | {"level":"info","time":"2021-07-22T10:38:06Z","message":"Migrator done"}
api_1       | panic: JWT_SIGNING_KEY was not set
api_1       | 
api_1       | goroutine 1 [running]:
api_1       | main.init.0()
api_1       | 	/app/cmd/api/auth.go:67 +0xc5
worker_1    | {"level":"info","time":"2021-07-22T10:38:06Z","message":"Starting worker"}
worker_1    | {"level":"error","error":"LOADING Redis is loading the dataset in memory","time":"2021-07-22T10:38:06Z","message":"Error when dequeuing message"}
updater_1   | {"level":"info","time":"2021-07-22T10:38:06Z","message":"Starting updater"}
redis_1     | 1:C 22 Jul 2021 10:38:06.096 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1     | 1:C 22 Jul 2021 10:38:06.096 # Redis version=6.2.3, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1     | 1:C 22 Jul 2021 10:38:06.096 # Configuration loaded
redis_1     | 1:M 22 Jul 2021 10:38:06.099 * monotonic clock: POSIX clock_gettime
redis_1     | 1:M 22 Jul 2021 10:38:06.103 * Running mode=standalone, port=6379.
redis_1     | 1:M 22 Jul 2021 10:38:06.103 # Server initialized
redis_1     | 1:M 22 Jul 2021 10:38:06.112 * <timeseries> RedisTimeSeries version 999999, git_sha=a7dedd78d585aa94f52c75b628c4daa61d3c544b
redis_1     | 1:M 22 Jul 2021 10:38:06.112 * <timeseries> Redis version found by RedisTimeSeries : 6.2.3 - oss
redis_1     | 1:M 22 Jul 2021 10:38:06.112 # <timeseries> could not initialize RedisGears_GetCompiledOs
redis_1     | 
redis_1     | 1:M 22 Jul 2021 10:38:06.113 # <timeseries> Failed initialize RedisGears API
redis_1     | 1:M 22 Jul 2021 10:38:06.113 * Module 'timeseries' loaded from /usr/lib/redis/modules/redistimeseries.so
redis_1     | 1:M 22 Jul 2021 10:38:06.115 * <module> Exported RedisJSON_V1 API
redis_1     | 1:M 22 Jul 2021 10:38:06.115 * <ReJSON> Created new data type 'ReJSON-RL'
redis_1     | 1:M 22 Jul 2021 10:38:06.115 * Module 'ReJSON' loaded from /usr/lib/redis/modules/rejson.so
redis_1     | 1:M 22 Jul 2021 10:38:06.119 * DB loaded from append only file: 0.004 seconds
redis_1     | 1:M 22 Jul 2021 10:38:06.119 * Ready to accept connections
```

**Comment by fnatte** [comment]
Hi, thanks for the issue and sorry for the late response. I'm still on annual leave.

You are correct about renaming `.env.default`. I've added that as a step in the README. The problem above seems to be that it is missing a value for `JWT_SIGNING_KEY`. 

My local .env is:

```
HOST=:8080
ORIGIN=http://localhost:3000
JWT_SIGNING_KEY=somesecret
``` 

I've updated the `.env.default` in the main branch, so you might want to fetch that or edit the file yourself.

Does that resolve the issue?
